book/accessibility.md

```diff
@@ -1355,11 +1351,12 @@ to the browser style sheet above:
 
 ``` {.python}
 class LineLayout:
-    def paint(self, display_list):
+    def paint_effects(self, cmds):
+        # ...
         for child in self.children:
             outline_str = child.node.parent.style.get("outline")
             if parse_outline(outline_str):
-                outline_rect.join(child.rect())
+                outline_rect.join(child.self_rect())
```

Comment:
So it looks like we added `self_rect` to every inline object?

Reply:
Yes. This was the code before?

Reply:
I think it's fine for the PR, but I will say that I'm becoming more and more convinced that we need to add a `Rect` class in Chapter 7 and then set up the expectation of each layout object having a `rect` method earlier.

Reply:
We did agree last week to add the rect class. :)